yt/frontends/enzo_p/io.py

```diff
@@ -162,7 +162,7 @@ def _read_obj_field(self, obj, field, fid_data):
         ftype, fname = field
         node = "/%s/field%s%s" % (obj.block_name, self._sep, fname)
         dg = h5py.h5d.open(fid, b(node))
-        rdata = np.empty(self.ds.grid_dimensions[:self.ds.dimensionality],
+        rdata = np.empty(self.ds.grid_dimensions[self.ds.dimensionality::-1],
```

Comment:
I'm interested to see what the answer tests say. For the 2D HelloWorld data, this would give you an array shape of [1,48,48], whereas it was originally [48,48].

Reply:
You're absolutely right. Now it doesn't work with the 2D HelloWorld test data. I'll look into it more.

Reply:
Okay, I think I fixed it in the latest commit.